workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/TransformSendSystem.cs

```diff
@@ -43,12 +45,11 @@ protected override void OnUpdate()
                 }
 
                 var entityId = transformData.SpatialEntityIds[i].EntityId;
-                var update = new global::Improbable.Transform.Transform.Update();
-                update.SetLocation(global::Generated.Improbable.Transform.Location.ToSpatial(component.Location));
-                update.SetRotation(global::Generated.Improbable.Transform.Quaternion.ToSpatial(component.Rotation));
-                update.SetTick(component.Tick);
-                Generated.Improbable.Transform.Transform.Translation.SendComponentUpdate(Worker.Connection, entityId,
-                    update);
+
+                var update = new SchemaComponentUpdate(component.ComponentId);
+                Generated.Improbable.Transform.SpatialOSTransform.Serialization.Serialize(component,
+                    update.GetFields());
+                worker.Connection.SendComponentUpdate(entityId, new ComponentUpdate(update));
```

Comment:
Was looking at random files for curiosity, should this be using the raw connection bits?

Reply:
Yes, currently. There's a ticket open to have helper methods for these.